src/functions.pl

```diff
@@ -428,6 +428,7 @@ sub promptFirstTimeInformation {
 
 sub processServerSettings {
 	my $filename = shift;
+	exit(0) if ($Settings::travis_ci);
```

Comment:
it cannot check error in pServer_servertype

Reply:
I think we need check with 
login -> char -> map -> wait 5 sec -> exit

Reply:
why wait 5 seconds?